contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.1.0--2.2.0.sql

```diff
@@ -1569,6 +1569,361 @@ INSERT INTO sys.babelfish_helpcollation VALUES (N'mongolian_cs_as', N'Mongolian,
 INSERT INTO sys.babelfish_helpcollation VALUES (N'sql_latin1_general_cp874_ci_as', N'Virtual, default locale, code page 874, case-insensitive, accent-sensitive, kanatype-insensitive, width-insensitive');
 INSERT INTO sys.babelfish_helpcollation VALUES (N'sql_latin1_general_cp874_cs_as', N'Virtual, default locale, code page 874, case-sensitive, accent-sensitive, kanatype-insensitive, width-insensitive');
 
+CREATE OR REPLACE FUNCTION sys.language()
+RETURNS sys.NVARCHAR(128)  AS 'babelfishpg_tsql' LANGUAGE C;
+
+CREATE OR REPLACE FUNCTION sys.host_name()
+RETURNS sys.NVARCHAR(128)  AS 'babelfishpg_tsql' LANGUAGE C IMMUTABLE PARALLEL SAFE;
+
+ALTER FUNCTION sys.tsql_stat_get_activity(text) RENAME TO tsql_stat_get_activity_deprecated_2_1;
+ALTER VIEW sys.sysprocesses RENAME TO sysprocesses_deprecated_2_1;
+
+-- deprecate views dependent on tsql_stat_get_activity
+ALTER VIEW sys.dm_exec_sessions RENAME TO dm_exec_sessions_deprecated_2_1;
+ALTER VIEW sys.dm_exec_connections RENAME TO dm_exec_connections_deprecated_2_1;
```

Comment:
why do we need this? Simply recreating these objects would be sufficient

Reply:
Won't objects already dependent on these views be affected if we simply re-create them?